tests/GridFieldOrderableRowsTest.php

```diff
@@ -49,6 +51,29 @@ public function testReorderItems() {
 
 	}
 
+	public function testSortableChildClass() {
+		$orderable = new GridFieldOrderableRows('Sort');
+		$reflection = new ReflectionMethod($orderable, 'executeReorder');
+		$reflection->setAccessible(true);
+		$parent = $this->objFromFixture('GridFieldOrderableRowsTest_Ordered', 'nestedtest');
+		$config = new GridFieldConfig_RelationEditor();
+		$config->addComponent($orderable);
+		$grid = new GridField(
+			'Children',
+			'Children',
+			$parent->Children(),
+			$config
+		);
+		$originalOrder = $parent->Children()->column('ID');
+		$desiredOrder = array_reverse($originalOrder);
+		$this->assertNotEquals($originalOrder, $desiredOrder);
+		$_REQUEST['showqueries'] = true;
+		$reflection->invoke($orderable, $grid, $desiredOrder);
+		$newOrder = $parent->Children()->column('ID');
+		unset($_REQUEST['showqueries']);
+		$this->assertEquals($desiredOrder, $newOrder);
```

Comment:
The formatting of this test method was much nicer in #158 ;)

Reply:
white space added - and removed the bad `$_REQUEST` access! yikes